### DISCUSSION TITLE
How to use Fireflyiii to account for assets?

### DISCUSSION THREAD

**Discussion by King-of-penguins** [firefly-iii/firefly-iii#4471] [Firefly III]
I have several bank deposits in different banks. I also have a brokerage account.
Once a few months I receive information about the accumulated amount of money in the bank account. And I have no information about how much money added to the bank account.
That is, I have the following data in some bank account (or some brokerage account):
23 June 2020 - 3000 Euro
12 November 2020 - 3029 Euro
2 March 2021 - 3058 Euro
So for each account in each bank.

On the basis of this data, I would like to be able to see: 
1) a chart of changing the amount of money in accounts 
2) relative rates of capital gains 
3) total amount of money on all accounts

Is this possible in the Fireflyiii program?

**Comment by niklas2810** [comment]
You can create a normal assert account for each bank. When you receive the info, you can then go into Firefly III and add the amount of money to the corresponding asset account as a transaction (with a message like " March 2021").

1: This should be possible using reports. For an individual account, you can go to the account page as well where a graph is displayed.
2: I'm not quite sure, but I think that this is out of scope of the software. However, you could write a simple script to display this data.
3: That's covered on the main page (dashboard).

This site might be interesting for you as well: https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/

**Reply by JC5** [reply]
10/10 points. :+1:

**Reply by King-of-penguins** [reply]
I think I did earlier what you wrote. If I add to an existing account of 3'000 euros a transaction of 3'029 euros, and then another transaction for 3'058 euros, then the final amount will be displayed: 9'087 euros. But the real amount should be 3'058 euros.
I could do differently. I could add to the account of 3'000 euros a transaction in 29 euros, and then another transaction of 29 euros (3'058-3'029), then will be the right amount to the account: 3058 euros.
Only in this case each time I need to calculate the added amount that is inconvenient.
I mean count:
1) 3029 Euro - 3000 Euro = 29 Euro
2) 3058 Euro - 3029 Euro = 29 Euro
It will need to be calculated manually (on the calculator).
